tests/inputData.json.example

```diff
@@ -57,6 +64,13 @@
 		"NSX Edge Nodes": [
 			"sfo-w01-en01",
 			"sfo-w01-en02"
+		],
+		"NSX Edge Fqdn": [
+			"sfo-w01-en01.sfo.rainpole.io",
+			"sfo-w01-en02.sfo.rainpole.io"
+		],
+		"Virtual Center": [
```

Comment:
```suggestion
		"vCenter Server": [
```

Reply:
Please replace instances of "Virtual Center" with "vCenter Server" in the JSON and the tests since that is the correct name.